resip/stack/ssl/Security.cxx

```diff
@@ -1913,19 +1938,41 @@ BaseSecurity::computeIdentity( const Data& signerDomain, const Data& in ) const
       return Data::Empty;
    }
 #endif
+#else // OPENSSL_VERSION_NUMBER < 0x30000000L
+   std::unique_ptr<EVP_PKEY_CTX, EVP_PKEY_CTX_deleter> pKeyCtx(EVP_PKEY_CTX_new(pKey, nullptr));
```

Comment:
From a quick glance the `EVP_PKEY` API is available since OpenSSL 1.0.2. If we drop support for earlier OpenSSL releases we could exclusively use the new code.